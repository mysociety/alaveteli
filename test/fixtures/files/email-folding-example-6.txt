Dear Mr. Brown,

Please find attached a reply to your FOI request.

Yours ever,

Adetokunbo Ighodaro

<<FOI 0169-08 final.doc>>

***********************************************************************************
Visit [1]http://www.fco.gov.uk for British foreign policy news and travel
advice; and [2]http://www.i-uk.com - the essential guide to the UK.

We keep and use information in line with the Data Protection Act 1998. We
may release this personal information to other UK government departments
and public authorities.

Please note that all messages sent and received by members of the Foreign
& Commonwealth Office and its
missions overseas may be monitored centrally. This is done to ensure the
integrity of the system.

***********************************************************************************

References

Visible links
1. http://www.fco.gov.uk/
2. http://www.i-uk.com/
