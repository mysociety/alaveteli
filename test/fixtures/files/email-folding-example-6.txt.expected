Dear Mr. Brown,

Please find attached a reply to your FOI request.

Yours ever,

Adetokunbo Ighodaro

<<FOI 0169-08 final.doc>>
FOLDED_QUOTED_SECTION
References

Visible links
1. http://www.fco.gov.uk/
2. http://www.i-uk.com/